FileTransactionStoreManagerTest.init:47 ExceptionInInitializer

- [ ] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description
```
Error:  io.seata.server.store.file.FileTransactionStoreManagerTest  Time elapsed: 1.097 s  <<< ERROR!
java.lang.ExceptionInInitializerError
	at io.seata.server.store.file.FileTransactionStoreManagerTest.init(FileTransactionStoreManagerTest.java:47)
Caused by: java.lang.NullPointerException: Cannot invoke "org.springframework.core.env.ConfigurableEnvironment.getProperty(String, java.lang.Class)" because "environment" is null
	at io.seata.server.store.file.FileTransactionStoreManagerTest.init(FileTransactionStoreManagerTest.java:47)
```

### Ⅱ. Describe what happened

  If there is an exception, please attach the exception trace:

```
Just paste your stack trace here!
```


### Ⅲ. Describe what you expected to happen


### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1. xxx
2. xxx
3. xxx

### Ⅴ. Anything else we need to know?


### Ⅵ. Environment:

- JDK version :
- Seata version:  
- OS :
- Others:

--- a/server/src/test/java/io/seata/server/session/GlobalSessionTest.java
+++ b/server/src/test/java/io/seata/server/session/GlobalSessionTest.java
@@ -21,11 +21,11 @@ import java.util.stream.Stream;
 import io.seata.core.model.BranchStatus;
 import io.seata.core.model.BranchType;
 import io.seata.core.model.GlobalStatus;
-import io.seata.server.storage.file.session.FileSessionManager;
 import io.seata.server.store.StoreConfig;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.MethodSource;
@@ -45,12 +45,12 @@ import static io.seata.common.DefaultValues.DEFAULT_TX_GROUP;
 public class GlobalSessionTest {
 
 
-    @BeforeAll
+    @BeforeEach
     public static void setUp(ApplicationContext context){
 
     }
     @BeforeAll
-    public static void init(){
+    public static void init(ApplicationContext context){
         SessionHolder.init(StoreConfig.SessionMode.FILE);
     }
     @AfterAll

--- a/server/src/test/java/io/seata/server/session/GlobalSessionTest.java
+++ b/server/src/test/java/io/seata/server/session/GlobalSessionTest.java
@@ -25,7 +25,6 @@ import io.seata.server.store.StoreConfig;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.MethodSource;
@@ -45,10 +44,6 @@ import static io.seata.common.DefaultValues.DEFAULT_TX_GROUP;
 public class GlobalSessionTest {
 
 
-    @BeforeEach
-    public static void setUp(ApplicationContext context){
-
-    }
     @BeforeAll
     public static void init(ApplicationContext context){
         SessionHolder.init(StoreConfig.SessionMode.FILE);

--- a/server/src/test/java/io/seata/server/store/file/FileTransactionStoreManagerTest.java
+++ b/server/src/test/java/io/seata/server/store/file/FileTransactionStoreManagerTest.java
@@ -35,6 +35,7 @@ import org.assertj.core.util.Files;
 import org.junit.jupiter.api.*;
 import org.mockito.Mockito;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.ApplicationContext;
 
 /**
  * @author ggndnn
@@ -43,7 +44,7 @@ import org.springframework.boot.test.context.SpringBootTest;
 public class FileTransactionStoreManagerTest {
 
     @BeforeAll
-    public static void init(){
+    public static void init(ApplicationContext context){
         SessionHolder.init(StoreConfig.SessionMode.FILE);
     }
     @AfterAll

